suite-native/transactions/src/utils.ts

```diff
@@ -42,3 +44,17 @@ export const sortTargetAddressesToBeginning = (
         A.intersection(addresses, targetAddresses),
         A.difference(addresses, targetAddresses),
     ) as VinVoutAddress[];
+
+export const getTransactionValueSign = (transactionType: TransactionType) => {
+    const signValueMap = {
```

Comment:
nitpick, this constant can be define outside the function to prevent redefinition on every function call...

Reply:
fixed: 59369acb4d